interfaces/Portalicious/src/locale/messages.nl.xlf

```diff
@@ -1066,7 +1066,22 @@
       <trans-unit id="3573847364103936169" datatype="html">
         <source>Note that if you go over 160 characters the Person affected may receive the message as multiple texts, if they have a feature-(non-smart-)phone.</source>
         <target state="new">Note that if you go over 160 characters the Person affected may receive the message as multiple texts, if they have a feature-(non-smart-)phone.</target>
+      <trans-unit id="6057281094654962766" datatype="html">
```

Comment:
```suggestion
      </trans-unit>
      <trans-unit id="6057281094654962766" datatype="html">
```

Missing closing tag is breaking the build (probably got lost in a conflict resolution). Same in the other xlf file.